src/test/java/org/candlepin/resource/OwnerResourceOwnerClaimingTest.java

```diff
@@ -212,6 +213,31 @@ public void claimantCannotBeAnonymous() {
             .isInstanceOf(BadRequestException.class);
     }
 
+    @Test
+    public void testClaimWhenClaimantExistOnlyUpstream() {
```

Comment:
Add Owner to the test name: `testClaimWhenClaimantOwnerExistsOnlyUpstream`

Reply:
done